cmd/vic-machine/create/create.go

```diff
@@ -876,14 +876,25 @@ func (c *Create) processVolumeStores() error {
 	defer trace.End(trace.Begin(""))
 	c.VolumeLocations = make(map[string]string)
 	for _, arg := range c.volumeStores {
-		if err := common.CheckUnsupportedCharsDatastore(arg); err != nil {
-			return fmt.Errorf("--volume-store contains unsupported characters: %s Allowed characters are alphanumeric, space and symbols - _ ( ) / :", err)
+		splitMeta := strings.Split(arg, ":")
+		if len(splitMeta) < 1 {
```

Comment:
agree with @hmahmood . suggest to move the logic of strings.Split here into the URL checking method, and return detailed error from that method.
I assume that method is used in parameter checking only.